### ISSUE TITLE
Tillana Medium diaereses above umlauts broken

### ISSUE THREAD

**Issue by bardware** [google/fonts#963]
Tillana Medium does not display German umlauts `äöü` correctly. Upper and lower case.

![image](https://cloud.githubusercontent.com/assets/802702/26130693/35c835fe-3a96-11e7-8c18-623b94f1b045.png)

**Comment by m4rc1e** [comment]
@bardware Thank you for spotting these interpolation errors :-). I'm a big fan of your screenshots.

@davelab6 shall we see if we can find all the problems with accents collections wide? This should be scriptable.

**Comment by bardware** [comment]
Love to help. Going through the fonts looking for the characters I care for is possible for me. Looking at the other charsets (kyrillic, vietnames etc.) is not.
If you script something to detect these errors: remember Istok https://github.com/google/fonts/issues/39

**Comment by laerm0** [comment]
Yeah, so maybe it's a design choice to have an dieresis with slightly-different dot sizes, but two of the five weights have matching dots and the other three (bold, medium, and semibold) have varying sizes. (Based on the glyphs master, it looks like they're supposed to be the same size and they got interpolated poorly in the instances.) I'm going to reach out to the designer for clarification.

![screen region 2017-08-03 at 15 34 37](https://user-images.githubusercontent.com/10780161/28940036-4d435d5e-7861-11e7-9b05-ecfa7ef18ab0.png)

**Comment by davelab6** [comment]
> it's a design choice

If so its the wrong one, lol :)

**Comment by laerm0** [comment]
It's been a month and I haven't heard back from the designer so, @davelab6, I agree with you: let's normalize these.

**Comment by laerm0** [comment]
Source on this doesn't have any instances set up. Anyone know of a way I can reverse-engineer them?

**Comment by davelab6** [comment]
Because @satyarajpurohit isn't fixing this, we will have to do it ourselves, and this will not be quick for us :( 

The sources are set up with afdko and a custom python build system "hindkit" which is public but the master branch is stale (see https://github.com/itfoundry/hindkit/branches) and that's why the instances aren't legible to you; they are set up in the python build system.
